packages/core/admin/index.js

```diff
@@ -304,6 +308,40 @@ async function watchFiles(dir) {
   });
 }
 
+const hasCustomAdminCode = async (dir) => {
+  const customAdminPath = path.join(dir, 'src/admin');
```

Comment:
Could we use `strapi.dirs.customAdmin` instead?

Reply:
we don't have a strapi instance at this moment